Projects/UOContent/Mobiles/BaseCreature.cs

```diff
@@ -2823,13 +2823,13 @@ public void AddSpellDefense(Type type)
         public Spell GetAttackSpellRandom()
         {
             var type = m_SpellAttack.RandomElement();
-            return type == null ? null : ActivatorUtil.CreateInstance(type, this, null) as Spell;
+            return type == null ? null : type.CreateInstance(this, null) as Spell;
         }
 
         public Spell GetDefenseSpellRandom()
         {
             var type = m_SpellDefense.RandomElement();
```

Comment:
Hmm should RandomElement() not always return a valid type and never null? then next line can be reduced to `return type.CreateInstance<Spell>(...)`